cluster-autoscaler/cloudprovider/azure/azure_config.go

```diff
@@ -65,6 +65,15 @@ type Config struct {
 	// It can override the default public ARM endpoint for VMs pool scale operations.
 	ARMBaseURLForAPClient string `json:"armBaseURLForAPClient" yaml:"armBaseURLForAPClient"`
 
+	// Managed system pool configuration for automatic cluster.
```

Comment:
Could we rename these to `HoboSubscriptionID`, `HoboResourceGroup`, and `HoboResourceProxyURL` to distinguish from the existing understanding of "managed" in the vanilla AKS sense?

(same comment for all the env vars, etc)

Reply:
Thanks @jackfrancis, I initially named them HoboXXX, but @tallaxes felt that it’s too specific to AKS internals so may not beideal for open-source usage. That’s why I went with the ManagedXXX naming. I'm open to suggestions @tallaxes

Reply:
How about just Hosted* for short?